services/src/test/scala/cromwell/pipeline/service/GitLabProjectVersioningTest.scala

```diff
@@ -183,7 +183,69 @@ class GitLabProjectVersioningTest extends AsyncWordSpec with ScalaFutures with M
           .getFile(project, path, Some(version))
           .map(_ shouldBe Left(VersioningException("Exception. Response status: 404")))
       }
+    }
 
+    "getFileCommits" should {
+      val dummyCommitJson: String = s"[${Json.stringify(Json.toJson(dummyFileCommit))}]"
```

Comment:
I think it will be more readable if you will do it like so: Json.stringify(Json.toJson(List(dummyFileCommit)))

Reply:
Yep, it's easier to use

Reply:
Changed